src/Module.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-mvc-middleware for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-mvc-middleware/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Mvc\Middleware;
+
+use Zend\Mvc\MiddlewareListener as DeprecatedMiddlewareListener;
+use Zend\ServiceManager\Factory\InvokableFactory;
+
+class Module
+{
+    /**
+     * @return array
+     */
+    public function getConfig()
+    {
+        return [
+            'service_manager' => [
+                'aliases' => [
+                    DeprecatedMiddlewareListener::class => MiddlewareListener::class,
```

Comment:
If the middleware listener is called "deprecated" here, should the class itself marked as deprecated? (The same for the controller.)

Reply:
@froschdesign It will be marked as deprecated in zend-mvc in next minor release. As well as deprecation notices added to the listener itself. But after this package is released.